flashcards/backend/decks/validators.py

```diff
@@ -0,0 +1,28 @@
+import os
+
+import magic
+from django.core.exceptions import ValidationError
+from django.core.files.base import ContentFile
+from django.core.files.storage import default_storage
+
+from config import settings
+
+
```

Comment:
we are handling here user upload validating only file type is not enough. I would look for size and dimentions validators.

Reply:
i agree, file size validator added